medical/models/res_partner.py

```diff
@@ -48,26 +32,31 @@ class ResPartner(models.Model):
     alias = fields.Char(
         string='Alias',
         size=256,
-        help='Common name that the Party is reffered',
+        help='Common name the party is referred to as',
     )
     activation_date = fields.Date(
-        string='Activation date',
-        help='Date of activation of the party',
+        string='Activation Date',
+        help='Date the party was activated',
     )
     last_name = fields.Char(
```

Comment:
We should remove this - it's not used anywhere & will create confusion.

Please add isolation of first and last names into the roadmap as well. It should probably be in patient, not partner too.